src/control/system/raft/raft_recovery_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/daos-stack/daos/src/control/common/test"
 	. "github.com/daos-stack/daos/src/control/lib/ranklist"
 	"github.com/daos-stack/daos/src/control/logging"
+	"github.com/daos-stack/daos/src/control/system"
 	. "github.com/daos-stack/daos/src/control/system"
```

Comment:
For clarity, want to get rid of the duplicate import?

Reply:
Ah, good call. I remember using the dot-imports when I moved stuff in order to reduce the reviewer burden, but now seems like a good time to get rid of them.